### ISSUE TITLE
Original code no longer returns intended output

### ISSUE THREAD

**Issue by QzSG** [prabhakar267/github-supported-languages#4]
Before the pull request for checking internet connection, your original code does not work. It simply returns 

```
[".."]
```

I would be doing a pull request to fix your code and to remove the redundant check that the other guy copy pasted from stackoverflow. There is no need to add `urllib` when requests does the same thing and is more supported. The other pull request and contribution is an outright spam just to get the hacktoberfest tshirt. No testing was even done to check if code is working as intended.

Also, I modified your screenshot to support cross platform compatability.

It is up to you to choose if you want to accept my push request later on

A developer should strive for correct code before adding extra modules and incomplete code.

**Comment by prabhakar267** [comment]
@QzSG I just saw PRs #2 #3 and it was wrong on my part to merge them. If you could drop those commits and modify your PR accordingly?
@ykl7

**Comment by QzSG** [comment]
@prabhakar267 If you see the pull request correctly you would see that I submitted it on top of #2 and #3, there is no need to drop the commits.

My initial pull request was before #2 and #3 was merged. I pulled in the changes before making pull request #5, dropping commits that have already been pulled to master is a bad practice and frowned upon (it can cause major havoc for contributors that have already pulled the changes to their local repos).

It would be fine and better to simply accept pr #5, cheers!
